app-misc/fff/fff-1.0.recipe

```diff
@@ -0,0 +1,35 @@
+SUMMARY="A simple file manager written in bash"
+DESCRIPTION="Features:
+* Fast
+* Minimal (only requires bash and coreutils)
+* Smooth Scrolling (using vim keybindings)
+* Supports LS_COLORS
+* File Oprations (copy, paste, rename, cut, etc)
+* Instant as you type search"
+HOMEPAGE="https://github.com/dylanaraps/fff"
+COPYRIGHT="2016-2018 Dylan Araps"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="6e5eed18b50b3e1a2497092e618665ff00b71b27b4f7653f9693ee40517d1a8a"
+SOURCE_FILENAME="fff-$portVersion.tar.gz"
+
+ARCHITECTURES="any"
+
+PROVIDES="
+	fff = $portVersion
+	cmd:fff
+	"
+REQUIRES="
+	haiku
+	bash
+	cmd:file
+	"
+
+INSTALL()
+{
+	install -d "$binDir" "$manDir"/man1 "$docDir"
+	install -m 755 fff "$binDir"
+	install -m 644 fff.1 "$manDir"/man1
+	install -m 644 {CONTRIBUTING,README}.md "$docDir"
```

Comment:
```suggestion
	install -m 644 README.md "$docDir"
```

The author has removed the `CONTRIBUTING` document right after release, so we should also do the same?